meta.py

```diff
@@ -27,5 +31,8 @@ def __init__(self):
     
     self._config = json.loads(json.dumps(self._config, default=str))
 
-  def get(self):
```

Comment:
You do not need a second function here, since it just returns a `dict`, You can simply make it `async`:
```python
async  def _get_config(self):
    return self._config
```